src/jumio-kyc/kyc.service.ts

```diff
@@ -363,7 +364,9 @@ export class KycService {
       }
 
       const allocatedOre = Math.floor((userPoolPoints / totalPoints) * poolOre);
-      await this.redemptionService.update(redemption, { [pool]: allocatedOre });
+      await this.redemptionService.update(redemption, {
```

Comment:
Do we need to use BigInt for all these?

Reply:
Max JS number is 2**53 - 1 and the amount of ORE in the genesis block is 42m * ORE_TO_IRON which is ~2**52 so seems safe that none of these numbers will reach an overflow